tests/bulk_create/tests.py

```diff
@@ -840,3 +843,71 @@ def test_update_conflicts_unique_fields_update_fields_db_column(self):
                 {"rank": 2, "name": "d"},
             ],
         )
+
+    @skipUnlessDBFeature("is_sqlite")
+    def test_bulk_create_with_where_clause_sqlite(self):
```

Comment:
These 2 tests appear to be identical … if so then they should be a single test and use

```
@skipUnlessDBFeature("supports_conditional_update")
```

Reply:
The tests may appear similar, but they are necessary to test the specific handling of the where_clause in the distinct SQLite and PostgreSQL backends